javalin/src/main/java/io/javalin/http/Context.kt

```diff
@@ -34,8 +35,8 @@ import javax.servlet.http.HttpServletResponse
 open class Context(@JvmField val req: HttpServletRequest, @JvmField val res: HttpServletResponse, internal val appAttributes: Map<String, Any> = mapOf()) {
 
     // @formatter:off
-    @get:JvmSynthetic @set:JvmSynthetic internal var inExceptionHandler = false
     @get:JvmSynthetic @set:JvmSynthetic internal var matchedPath = ""
+    @get:JvmSynthetic @set:JvmSynthetic internal var inExceptionHandler = false
```

Comment:
*Quick info - I had to swap these fields because I was unable to compile project locally (NoSuchMethodErrors), I'll swap it in the last commit for this PR after review